cmd/collectors/rest/plugins/shelf/shelf.go

```diff
@@ -316,6 +318,14 @@ func (my *Shelf) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 		return true
 	})
 
+	if len(noSet) > 0 {
+		attributes := make([]string, 0)
+		for k := range noSet {
+			attributes = append(attributes, k)
+		}
```

Comment:
maps.Keys(noSet) from golang.org/x/exp/maps?

Reply:
I thought about that but give the small amount of code needed, it seems better to stick with our own keys instead of pulling in a experimental package.

> Warning: Packages here are experimental and unreliable. Some may one day be promoted to the main repository or other subrepository, or they may be modified arbitrarily or even disappear altogether.

> In short, code in this subrepository is not subject to the Go 1 compatibility promise. (No subrepo is, but the promise is even more likely to be violated by go.exp than the others.)

Reply:
Got it.